test/cli.js

```diff
@@ -2,25 +2,97 @@
 
 // Node.js built-ins
 
+const os = require('os');
 const path = require('path');
 
 // foreign modules
 
 const execa = require('execa');
+const fs = require('graceful-fs');
 const npmBinTester = require('npm-bin-ava-tester');
 const test = require('ava');
 
 // local modules
 
+const lib = require('../lib/commands.js');
+
+// this module
+
+function copyExeSync (source /* : string */, destination /* : string */) {
```

Comment:
Probably dropped it when those ava FlowType things were going berserk. I'll double-check this.